src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,107 @@
+package duke.storage;
+
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Storage {
+    private final Path filePath;
+
+    public Storage() {
+        this.filePath = Paths.get(System.getProperty("user.dir"), "data.txt");
+    }
+
+    /**
+     * Returns a Task object that has been deserialized from its string representation.
```

Comment:
There should be an empty line between the Javadocs method description and the params.

Reply:
I see. Thank you for the input. I will update my code accordingly 👍